how-to-guides/re-run-failed-training/scripts/re_run_failed_training.py

```diff
@@ -0,0 +1,113 @@
+import neptune.new as neptune
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import os
+from torchvision import datasets, transforms
+
+# Step 1: Get Run ID
+
+# Fetch project
+project = neptune.get_project(name='common/showroom', api_token='ANONYMOUS')
+
+# Fetch only inactive runs
+runs_table_df = project.fetch_runs_table(state="idle", tag=['showcase-run']).to_pandas()
+
+# Sort runs by failed 
+runs_table_df = runs_table_df.sort_values(by='sys/failed', ascending=True)
+
+# Extract the last failed run's id
+failed_run_id = runs_table_df[runs_table_df['sys/failed']==True]['sys/id'].values[0]
+
+print('Failed_run_id = ', failed_run_id)
+
+# Step 2: Resume failed run
+failed_run = neptune.init(
```

Comment:
Please keep order of arg as in the docs: https://docs.neptune.ai/api-reference/neptune#.init_run
It's a bit easier for the user to read code here and the docs.